units/console-getty.service.m4.in

```diff
@@ -16,7 +16,7 @@ After=rc-local.service
 Before=getty.target
 
 [Service]
-ExecStart=-/sbin/agetty --noclear --keep-baud console 115200,38400,9600 $TERM
+ExecStart=-/sbin/agetty --login-options='-p -- \u' --noclear --keep-baud console 115200,38400,9600 $TERM
```

Comment:
I think `\u` should be escaped (as `\\u`). systemd.service(5) says "Only escape patterns which match the syntax in the table are allowed; other patterns may be added in the future and unknown patterns will result in a warning. In particular, any backslashes should be doubled.".

Also, please add a brief explanation in a comment how this `--login-options` option is interpreted. I know that one should look at the man page, but this is pretty unusual syntax, so a short explanation is in order.